decidim-proposals/app/presenters/decidim/proposals/admin_log/value_types/proposal_title_body_presenter.rb

```diff
@@ -5,10 +5,13 @@ module Proposals
     module AdminLog
       module ValueTypes
         class ProposalTitleBodyPresenter < Decidim::Log::ValueTypes::DefaultPresenter
+          include Decidim::TranslatableAttributes
+
           def present
             return unless value
 
-            renderer = Decidim::ContentRenderers::HashtagRenderer.new(value)
+            translated_value = translated_attribute(value)
```

Comment:
maybe it is important to take the following into account?
```suggestion
            translated_value = translated_attribute(value)
            return unless translated_value.present?

```

Reply:
I think the `translated_attribute` method already handles that but yeah, probably better safe than sorry 😄